cookbooks/active_dau.md

```diff
@@ -0,0 +1,30 @@
+# Active DAU
+
+An "Active User" is defined as a client who has total_daily_uri >= 5 URI for a given date. 
+* Dates are defined by `submission_date_s3`.
+* A client's total_daily_uri is defined as their sum of `scalar_parent_browser_engagement_total_uri_count` for a given date[1]. 
+
+Here is an example query in `main_summary` for getting Active DAU (aDAU) over time: 
```

Comment:
DAU and aDAU are both triggering spelling errors, but should not be. Can you please add these terms to the .spelling file in the root of the repository?

Reply:
`clients_daily_v6` is out as of last week and this is already a pre-computed field in the dataset: https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/ClientsDailyView.scala#L218

I'd encourage using that dataset over main_summary